src/App.js

```diff
@@ -1,73 +1,191 @@
-import React from 'react'
-import { Experiment, jsPsych } from 'jspsych-react'
-import { tl } from './timelines/main'
-import { config } from './config/main'
-import './App.css'
-import 'bootstrap/dist/css/bootstrap.css'
-import '@fortawesome/fontawesome-free/css/all.css'
-import { getTurkUniqueId, sleep } from './lib/utils'
-import { addToFirebase, createFirebaseDocument } from './firebase.js'
+import React, { useState, useEffect, useCallback } from "react";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.css";
+import "@fortawesome/fontawesome-free/css/all.css";
+import Login from "./components/Login";
+import { jsPsych } from "jspsych-react";
```

Comment:
group the imports by type (external libraries, then css, then components, then internal functions)